<launch>
 <!--node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="-7.00587 1.38311 3.64586 -0.1359765 0.9624515 0.0111864 0.2346749 world camera_link 100" /-->
 <!--node pkg="tf" type="static_transform_publisher" name="link2_broadcaster" args="-4.79288635 2.15943100 -0.71797882 0.0219252 -0.0017217 0.7804572 0.6248223 world base_link 100" /-->
  <arg name="node_namespace" default="apriltag_ros_offline_node" />
  <arg name="camera_frame" default="camera_link" />
  <arg name="image_topic" default="/image_raw" />
  <arg name="camera_name" default="/videofile" />

  <!-- Set parameters -->
  <rosparam command="load" file="$(find apriltag_ros)/config/settings.yaml" ns="$(arg node_namespace)" />
  <rosparam command="load" file="$(find apriltag_ros)/config/tags.yaml" ns="$(arg node_namespace)" />
  <node pkg="apriltag_ros" type="apriltag_ros_offline_node" name="$(arg node_namespace)" clear_params="true" output="screen" >

    <!-- Remap topics from those used in code to those on the ROS network -->
    <remap from="image_rect" to="$(arg camera_name)/$(arg image_topic)" />
    <remap from="camera_info" to="$(arg camera_name)/camera_info" />
    <param name="video_file" type="string" value="/mnt/d/video/0730/VID_20200730_160802.mp4"/>
    <param name="output_file" type="string" value="/mnt/d/video/0730/20200730_160802.txt"/>
    <param name="camera_frame" type="str" value="$(arg camera_frame)" />
    <param name="publish_tag_detections_image" type="bool" value="false" />      <!-- default: false -->
  </node>
</launch>
